Update iOS page with USBC and further information (#982)
jamulussoftware/jamulussoftware.github.io: wiki/en/Installation-for-iOS.md

* To install Jamulus on your iOS device, you need a (free or paid) Apple Developer account and preferably a computer.
* If you don't pay for an Apple developer account, Jamulus will only run for at most 7 days before you need to re-install it.
* Jamulus on iOS works quite well if everything is set up correctly. However, we strongly recommend you use an audio interface and an USB-Ethernet adapter. If you own an older device which has a Lightning port and not USB-C, you can use something like the [Lightning to USB Camera Adapter](https://www.apple.com/uk/shop/product/MD821ZM/A/lightning-to-usb-camera-adapter) and a (powered) USB-hub.

> * If use Jamulus on a wireless network, you should prefer a cellular (5G) connection over WiFi.

Reviewer: ```suggestion
* If you must use Jamulus on a wireless network, you should prefer a cellular (5G) connection over WiFi.
```

> * If you must use Jamulus on a wireless network, you should prefer a cellular (5G) connection over WiFi.

Reviewer: Is this iOS-specific?

Author: I don't know about android but from experience with a 5G router + Ethernet + Linux, cellular quality is much better compared to WiFi. 
I'd say it's not OS specific.

Reviewer: It's probably better left off this page, or at least make it a reference to the general guidance.  (Maybe somewhere like hints and tips would be better).

Author: I think it's mainly a mobile thing. Thus it's here (computers with 5G exist, but are rare)

Reviewer: > I think it's mainly a mobile thing. 

So we should have a "mobile" page with separate sections for Andriod and iOS specifics, rather than two separate pages?  If we've information that's pertinent to more than one platform, it should be somewhere common to those platforms so it gets managed as a single item.

Author: Maybe put it on the FAQ page if it isn't already? I wouldn't like to maintain a separate mobile page. That adds too much clutter.

Reviewer: We have two pages.

I suggested one page in their place.

That's less "clutter", right?

Author: I understood that differently: 1 page per OS and then one page for mobile.

I think having a page per OS is sensible as installation works differently. We can use imports to show text on multiple pages. @gilgongo did that in the past, I think.

So this import would then discuss about mobile.

Reviewer 2: Yes, duplication of content is the main problem to avoid. So we have used includes in the past for sharing the same info between pages (if we think it may need changing in the future). 

Pages per OS seems OK to me, since most people will want to read stuff that presents itself for their OS I think.

## Feedback and development

> iOS support isn't mature and not ready to be published on the AppStore. Feel free to help out:

Reviewer: ```suggestion
The view of the Jamulus development team is that the iOS version is not sufficiently mature to be published on the AppStore. Feel free to help out:
```

Author: Why not "our" view?

Reviewer 2: That would be shorter :-)

Reviewer: So?
```suggestion
Our view is that the iOS version is not sufficiently mature to be published on the AppStore. Feel free to help out:
```
Is it clear to the reader to whom "our" refers?

Reviewer 2: I think so. I think I've seen other F/LOSS projects using "our" to refer to the people who maintain the project.
